### PR TITLE
HHbbggExtendedTraining

### PR DESCRIPTION
**Pull request by ivovtin**
Dear flashgg developers,
here added extra variables for Training in HHbbgg analysis.

### CODE REVIEW

**Comment by simonepigazzini** [comment]
@chernyavskaya please review.

**Review by michelif — APPROVED** [review]
ciao, everything looks ok to me.

**Comment by nadya-chernyavskaya** [comment]
But why do you need to create a separate MVA ? it does not make any sense to have two MVAs. Please only add the variables everywhere. We should only have one MVA in the end seith the extended variables. Also please put the scaling back as it was before. if you do not want to use it , you simply set the value to 1. in the config file.

File: MetaData/data/MetaConditions/Era2016_RR-17Jul2018_v1.json [hunk 1]
```diff
@@ -118,7 +118,9 @@
 	"jetID" : "Loose",
 	"weightsFile" : "flashgg/Taggers/data/HHTagger/training_with_19_03_2019_trainingMjj_year0.weights.xml",
 	"MVAFlatteningFileName" : "flashgg/Taggers/data/HHTagger/cumulativeTransformation_20190321_2016_2017.root",
-	"MVAscalingValue" : 1.0,
+        "MVAscalingValue" : 1.0,
```
Comment (simonepigazzini) [inline MetaData/data/MetaConditions/Era2016_RR-17Jul2018_v1.json:121]: please make the indentation uniform with the rest of the file

**Comment by simonepigazzini** [comment]
Dear Ivan,

I think Nadya's comment about having a single MVA makes sense. Instead of duplicating the code for MVAExtedend please simply put the additional variables in the original MVAComputer and adjust the other variables if needed. Also please leave the option for the scaling in place and check the indentation in the json files.

Thank you,

simone

**Comment by ivovtin** [comment]
Thanks for comments. I'll fix it

**Comment by simonepigazzini** [comment]
wait, thinking again I think it's fine to have the two separate implementations. We can clean up later on.
For the moment please just fix the indentation in the json file and I will merge it as it is.

Best,

simone

**Comment by ivovtin** [comment]
Ok. I try check indentation, but here looks good: https://github.com/ivovtin/flashgg/blob/dev_legacy_runII/MetaData/data/MetaConditions/Era2016_RR-17Jul2018_v1.json

**Comment by simonepigazzini** [comment]
ok probably it's just an visualization issue on the PR page, thank you for checking. We will see if it makes more sense to merge the PR directly of merge it together with the other one opened by Nadya.

**Comment by simonepigazzini** [comment]
I'm closing this PR since the relevant code and commits were imported in PR #1182.

**Pull request closed without merging**